components/toast_wrapper/toast_wrapper.tsx

```diff
@@ -130,7 +122,7 @@ class ToastWrapper extends React.PureComponent {
             showUnreadToast = true;
         }
 
-        if (!showUnreadToast && unreadCount > 0 && !props.atBottom && (props.lastViewedBottom < props.latestPostTimeStamp)) {
+        if (!showUnreadToast && unreadCount && unreadCount > 0 && !props.atBottom && props.latestPostTimeStamp && (props.lastViewedBottom < props.latestPostTimeStamp)) {
```

Comment:
you don't need to check twice for unreadCount, if its failing we can ensure this won't return undefined by passing a default value. You can also shorten the condition ```(props?.lastViewedBottom < props?.latestPostTimeStamp))```

Reply:
Thanks for the suggestion. I checked again. It seems like `unreadCount` will never be undefined. Therefore, I removed the additional checks